centreon/tests/e2e/features/Backup-configuration/01-backup-properties/index.ts

```diff
@@ -71,7 +71,7 @@ Then('the MySQL configuration file path is set', () => {
     cy.get('input[name="backup_configuration_files"]').should('be.checked');
     cy.get('input[name="backup_mysql_conf"]').should(
       'have.value',
-      '/etc/my.cnf.d/centreon.cnf'
+      ''
     );
```

Comment:
```suggestion
cy.get('input[name="backup_mysql_conf"]').should('have.value', '');
```

I think linting should only run on a single line ?